### PR TITLE
Change error message names to product

### PR DESCRIPTION
**Pull request by emlyn-rice**
Fix path in executor

### CODE REVIEW

File: .changeset/selfish-snails-think.md
Comment (idadaniels) [inline .changeset/selfish-snails-think.md:1]: Is this changeset related to this PR?

**Review by idadaniels — APPROVED** [review]

File: .changeset/selfish-snails-think.md
Comment (emlyn-rice) [reply .changeset/selfish-snails-think.md:1]: It's missed from the previous one. ;(
